samples/snippets/create_job_with_embedded_captions.py

```diff
@@ -0,0 +1,199 @@
+                        bitrate_bps=550000,
+                        frame_rate=60,
+                    ),
+                ),
+            ),
+            transcoder_v1.types.ElementaryStream(
+                key="audio-stream0",
+                audio_stream=transcoder_v1.types.AudioStream(
+                    codec="aac", bitrate_bps=64000
+                ),
+            ),
+            transcoder_v1.types.ElementaryStream(
+                key="cea-stream0",
+                # The following doesn't work because "mapping" is a reserved
+                # argument name in GCP python client libraries (see
+                # https://github.com/googleapis/proto-plus-python/blob/main/proto/message.py#L447):
+                #
+                # text_stream=transcoder_v1.types.TextStream(
+                #   codec="cea608",
+                #   mapping=[
+                #     transcoder_v1.types.TextStream.TextMapping(
+                #       atom_key="atom0",
+                #       input_key="caption-input0",
+                #       input_track=0,
+                #     ),
+                #   ],
+                # ),
+                # Use a python dictionary as a workaround:
+                text_stream={
+                    "codec": "cea608",
+                    "mapping": [
+                        {
+                            "atom_key": "atom0",
+                            "input_key": "caption-input0",
+                            "input_track": 0,
+                        }
+                    ],
+                },
+            ),
+        ],
+        mux_streams=[
+            transcoder_v1.types.MuxStream(
+                key="sd",
+                container="mp4",
+                elementary_streams=["video-stream0", "audio-stream0"],
+            ),
+            transcoder_v1.types.MuxStream(
+                key="sd-hls",
+                container="ts",
+                elementary_streams=["video-stream0", "audio-stream0"],
+            ),
+            transcoder_v1.types.MuxStream(
+                key="sd-dash",
+                container="fmp4",
+                elementary_streams=["video-stream0"],
+            ),
+            transcoder_v1.types.MuxStream(
+                key="audio-dash",
+                container="fmp4",
+                elementary_streams=["audio-stream0"],
+            ),
+        ],
+        manifests=[
+            transcoder_v1.types.Manifest(
+                file_name="manifest.m3u8",
+                # In a similar situation to `mapping`, the autogenerated
+                # library added _ to the end of the reserved word `type`.
+                type_="HLS",
+                mux_streams=["sd-hls"],
+            ),
+            transcoder_v1.types.Manifest(
+                file_name="manifest.mpd",
+                # In a similar situation to `mapping`, the autogenerated
+                # library added _ to the end of the reserved word `type`.
```

Comment:
I think it's okay to omit this explanation here since it's close to the one above.

Reply:
Done